packages/cli/src/config.ts

```diff
@@ -51,3 +51,5 @@ export const jsonToGlobalConfig = (json: unknown) => {
 };
 
 export type GlobalConfig = z.infer<typeof zGlobalConfig>;
+
+export const PROJECT_TEMPALTES = ["vercel", "netlify", "netlify-functions"];
```

Comment:
Were both functions? The distinction was in "edge", no?

Reply:
Yes, and i think the decisions was `edge` is cheaper and efficient make it default. We can add the differences between these in docs.

Reply:
I mean netlify-functions does not describe at all. Both netlify options are functions

Reply:
Switched back to `netlify-functions` and `netlify-edge-functions` from their platform